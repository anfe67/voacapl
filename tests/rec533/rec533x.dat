OUTPUT        6    0
COEFFS    CCIR
SUNSPOT    100.
TIME          1   24    1
MONTH      1994    6
TRANS     TANGIER, Morocco      35.80N      5.90W  S     D
RECVR     BELGRADE              44.90N     20.50E  S     D
SYST          1.0000 3.00    0  0.0
ANTENNA       1    1    2   30     0.000[DEFAULT\CONST17.VOA  ]  0.0  500.0000
ANTENNA       2    2    2   30     0.000[DEFAULT\SWWHIP.VOA   ]  0.0    0.0000
NOIS       -145    1   73   90
FREQ        6.075  7.200  9.700 11.850 13.700 15.350 17.725 21.650 25.885  0.000
EXEC
QUIT
