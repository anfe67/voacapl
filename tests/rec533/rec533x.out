 CCIR Coefficients  METHOD  6   REC533 Version 16.1207a             PAGE   1
                                                  Analog modulation
 JUN    1994          SSN = 100.                      Path
 TANGIER, Morocco    BELGRADE               AZIMUTHS <Short> N. MI.      KM
 35.80 N    5.90 W   44.90 N   20.50 E     57.41  254.73    1318.1   2440.9
 MIN ANG  3.0 DEG
 XMTR  2-30 2-D Table [DEFAULT\CONST17.VOA  ] Az=  0.0 OFFaz= 57.4 500.000kW
 RCVR  2-30 2-D Table [DEFAULT\SWWHIP.VOA   ] Az=  0.0 OFFaz=254.7
 NOISE -145 dBW           S/N 90% of Days @  73 dB  in    1 Hz RX Bandwidth
 
 UT  MUF                                                         LUF  FOT OPMUF
 
  1 15.3  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  2.9 15.0  19.1
     1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      16    4    5    7    9   12   16   16   17   18    0 ANGL
      69   71   71   70   69   69   66   54   46   39 -999 DBU
      58   66   65   62   59   58   55   42   32   23 -999 dBpW
     102   95   96   99   99  101  100   88   80   74 -999 S/N
    0.99 0.99 0.99 0.99 0.99 0.99 0.98 0.86 0.74 0.55 0.00 FS/N
      84   81   82   85   86   84   82   70   64   59 -999 SNxx
 
  2 14.3  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  2.6 14.1  17.9
     1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      16    5    5    8   10   14   16   17   18   19    0 ANGL
      69   71   71   70   69   69   59   51   43   36 -999 DBU
      58   66   64   62   59   58   48   39   29   21 -999 dBpW
     102   96   97   99  100  101   92   85   78   72 -999 S/N
    0.99 0.99 0.99 0.99 0.99 0.99 0.92 0.82 0.69 0.42 0.00 FS/N
      85   82   83   86   83   84   75   67   62   56 -999 SNxx
 
  3 14.1  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  2.6 12.7  16.2
     1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      15    4    5    7   10   14   16   16   17   18    0 ANGL
      69   71   71   70   69   69   58   51   43   36 -999 DBU
      58   66   64   61   59   58   47   38   29   20 -999 dBpW
     101   97   98   98  100  101   91   84   77   71 -999 S/N
    0.99 0.99 0.99 0.99 0.99 0.99 0.91 0.81 0.67 0.39 0.00 FS/N
      85   82   83   85   83   85   74   66   62   56 -999 SNxx
 
  4 15.0  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  4.5 13.5  17.3
     1F2  2F2  2F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      14   17   21    6    8   11   14   15   15   16    0 ANGL
      69   65   66   69   68   69   63   53   45   38 -999 DBU
      57   62   61   61   58   58   52   41   31   23 -999 dBpW
     101   94   95   98   99  100   96   87   79   73 -999 S/N
    0.99 0.97 0.98 0.99 0.99 0.99 0.95 0.85 0.72 0.52 0.00 FS/N
      84   79   81   84   85   84   78   68   63   58 -999 SNxx
 
  5 16.8  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  6.3 15.0  19.3
     1F2  2F2  2F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      11   26   26   11   11   11   11   11   11   11    0 ANGL
      68   58   62   67   67   67   68   60   49   42 -999 DBU
      55   55   57   59   57   56   56   46   34   26 -999 dBpW
     101   87   91   97   98   99  100   93   83   77 -999 S/N
    0.99 0.89 0.94 0.99 0.99 0.99 0.98 0.92 0.78 0.65 0.00 FS/N
      83   72   76   83   84   82   82   75   64   61 -999 SNxx
CCIR Coefficients  METHOD  6   REC533 Version 16.1207a             PAGE   2
                                                  Analog modulation
 JUN    1994          SSN = 100.                      Path
 TANGIER, Morocco    BELGRADE               AZIMUTHS <Short> N. MI.      KM
 35.80 N    5.90 W   44.90 N   20.50 E     57.41  254.73    1318.1   2440.9
 MIN ANG  3.0 DEG
 XMTR  2-30 2-D Table [DEFAULT\CONST17.VOA  ] Az=  0.0 OFFaz= 57.4 500.000kW
 RCVR  2-30 2-D Table [DEFAULT\SWWHIP.VOA   ] Az=  0.0 OFFaz=254.7
 NOISE -145 dBW           S/N 90% of Days @  73 dB  in    1 Hz RX Bandwidth
 
 UT  MUF                                                         LUF  FOT OPMUF
 
  6 18.8  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  8.1 16.4  21.6
     1F2  2E   2F2  2F2  1F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      10    7   25   25   10   10   10   10   10   10    0 ANGL
      67   45   53   60   65   65   66   67   54   46 -999 DBU
      53   41   48   53   55   54   54   53   39   29 -999 dBpW
     100   74   83   91   96   97   98  100   87   80 -999 S/N
    0.98 0.55 0.82 0.95 0.99 0.99 0.97 0.98 0.85 0.74 0.00 FS/N
      82   59   68   77   82   83   80   82   69   64 -999 SNxx
 
  7 20.0  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  9.9 17.3  23.0
     1F2  2E   2E   2F2  2F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      11    7    7   25   25   11   11   11   11   11    0 ANGL
      66   34   42   55   58   62   64   65   56   48 -999 DBU
      52   29   36   48   49   51   52   52   41   31 -999 dBpW
      99   63   71   86   91   94   96   98   90   82 -999 S/N
    0.98 0.08 0.37 0.89 0.94 0.98 0.98 0.97 0.89 0.76 0.00 FS/N
      82   47   56   72   76   80   81   80   72   63 -999 SNxx
 
  8 20.1  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ 11.6 17.4  23.1
     1F2  2E   2E   2F2  2F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      11    7    7   26   26   11   11   11   11   11    0 ANGL
      65   24   33   50   55   59   62   64   56   48 -999 DBU
      51   19   27   43   46   48   50   51   41   31 -999 dBpW
      99   53   63   82   88   91   94   97   90   82 -999 S/N
    0.97 0.01 0.08 0.81 0.91 0.95 0.96 0.96 0.89 0.76 0.00 FS/N
      81   37   48   68   74   77   79   79   72   63 -999 SNxx
 
  9 19.8  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ 12.6 17.2  22.7
     1F2  2E   2E   2F2  2F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      13    7    7   28   28   13   13   13   13   13    0 ANGL
      65   16   26   47   53   57   60   63   54   46 -999 DBU
      51   11   20   39   44   46   49   50   40   30 -999 dBpW
      98   44   56   78   85   89   93   96   88   81 -999 S/N
    0.97 0.01 0.01 0.72 0.87 0.93 0.95 0.95 0.87 0.76 0.00 FS/N
      80   29   40   64   71   75   78   78   70   64 -999 SNxx
 
 10 19.6  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ 13.3 17.3  22.5
     1F2  2E   2E   2F2  2F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      13    7    7   29   29   13   13   13   13   13    0 ANGL
      64   10   21   44   52   56   60   63   54   46 -999 DBU
      50    5   15   37   43   45   48   50   39   30 -999 dBpW
      98   38   50   76   84   88   92   96   88   81 -999 S/N
    0.96 0.01 0.01 0.62 0.85 0.91 0.95 0.95 0.85 0.75 0.00 FS/N
      80   23   35   61   70   74   77   78   69   64 -999 SNxx
CCIR Coefficients  METHOD  6   REC533 Version 16.1207a             PAGE   3
                                                  Analog modulation
 JUN    1994          SSN = 100.                      Path
 TANGIER, Morocco    BELGRADE               AZIMUTHS <Short> N. MI.      KM
 35.80 N    5.90 W   44.90 N   20.50 E     57.41  254.73    1318.1   2440.9
 MIN ANG  3.0 DEG
 XMTR  2-30 2-D Table [DEFAULT\CONST17.VOA  ] Az=  0.0 OFFaz= 57.4 500.000kW
 RCVR  2-30 2-D Table [DEFAULT\SWWHIP.VOA   ] Az=  0.0 OFFaz=254.7
 NOISE -145 dBW           S/N 90% of Days @  73 dB  in    1 Hz RX Bandwidth
 
 UT  MUF                                                         LUF  FOT OPMUF
 
 11 19.7  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ 13.8 17.7  22.6
     1F2  2E   2E   2F2  2F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      14    7    7   29   29   14   14   14   14   14    0 ANGL
      64    6   18   43   51   55   59   62   54   46 -999 DBU
      50    1   12   35   42   44   47   49   39   30 -999 dBpW
      97   35   47   74   83   87   92   95   88   81 -999 S/N
    0.96 0.01 0.01 0.55 0.82 0.90 0.94 0.95 0.86 0.75 0.00 FS/N
      80   20   32   60   68   73   76   78   70   64 -999 SNxx
 
 12 19.8  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ 13.8 17.8  22.8
     1F2  2E   2E   2F2  2F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      14    7    7   29   29   14   14   14   14   14    0 ANGL
      64    6   18   43   50   55   59   62   54   46 -999 DBU
      50    1   12   35   41   44   47   49   40   30 -999 dBpW
      97   34   47   74   82   87   91   95   88   81 -999 S/N
    0.96 0.01 0.01 0.54 0.82 0.90 0.94 0.95 0.87 0.76 0.00 FS/N
      80   19   32   60   68   73   76   77   70   64 -999 SNxx
 
 13 19.8  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ 13.4 17.7  22.7
     1F2  2E   2E   2F2  2F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      13    7    7   29   29   13   13   13   13   13    0 ANGL
      64    9   21   44   51   56   59   62   54   46 -999 DBU
      50    4   15   37   42   45   48   50   40   30 -999 dBpW
      98   37   50   75   83   88   92   96   88   81 -999 S/N
    0.96 0.01 0.01 0.61 0.83 0.91 0.94 0.95 0.87 0.76 0.00 FS/N
      80   22   35   61   69   73   77   78   70   64 -999 SNxx
 
 14 19.5  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ 12.9 17.5  22.4
     1F2  2E   2E   2F2  2F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      13    7    7   29   29   13   13   13   13   13    0 ANGL
      64   15   26   47   53   57   61   63   54   46 -999 DBU
      51   10   20   40   44   47   49   51   39   30 -999 dBpW
      98   43   55   78   85   89   93   96   88   81 -999 S/N
    0.97 0.01 0.01 0.70 0.86 0.93 0.95 0.95 0.85 0.75 0.00 FS/N
      80   28   40   63   70   75   78   79   69   64 -999 SNxx
 
 15 19.4  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ 12.4 17.5  22.3
     1F2  2E   2E   2F2  2F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      13    7    7   29   29   13   13   13   13   13    0 ANGL
      65   23   34   51   55   60   62   64   53   46 -999 DBU
      51   18   28   43   46   49   51   51   39   30 -999 dBpW
      98   51   63   81   86   91   94   97   87   80 -999 S/N
    0.97 0.01 0.09 0.78 0.89 0.95 0.96 0.96 0.85 0.75 0.00 FS/N
      81   35   47   66   72   77   79   79   69   64 -999 SNxx
CCIR Coefficients  METHOD  6   REC533 Version 16.1207a             PAGE   4
                                                  Analog modulation
 JUN    1994          SSN = 100.                      Path
 TANGIER, Morocco    BELGRADE               AZIMUTHS <Short> N. MI.      KM
 35.80 N    5.90 W   44.90 N   20.50 E     57.41  254.73    1318.1   2440.9
 MIN ANG  3.0 DEG
 XMTR  2-30 2-D Table [DEFAULT\CONST17.VOA  ] Az=  0.0 OFFaz= 57.4 500.000kW
 RCVR  2-30 2-D Table [DEFAULT\SWWHIP.VOA   ] Az=  0.0 OFFaz=254.7
 NOISE -145 dBW           S/N 90% of Days @  73 dB  in    1 Hz RX Bandwidth
 
 UT  MUF                                                         LUF  FOT OPMUF
 
 16 19.7  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ 11.6 17.7  22.6
     1F2  2E   2E   2F2  2F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      12    7    7   28   28   12   12   12   12   12    0 ANGL
      65   32   42   54   58   62   63   65   55   46 -999 DBU
      51   27   36   47   48   51   51   52   40   30 -999 dBpW
      99   59   70   84   88   92   94   97   88   81 -999 S/N
    0.97 0.04 0.31 0.84 0.91 0.96 0.97 0.96 0.86 0.76 0.00 FS/N
      81   44   55   69   73   78   79   79   70   64 -999 SNxx
 
 17 20.4  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  9.6 18.4  23.5
     1F2  2E   2F2  2F2  1F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      11    7   26   26   11   11   11   11   11   11    0 ANGL
      66   43   51   59   64   65   65   66   58   49 -999 DBU
      52   39   46   52   54   54   53   53   43   32 -999 dBpW
     100   70   79   88   93   95   96   98   91   83 -999 S/N
    0.98 0.30 0.72 0.90 0.97 0.99 0.99 0.97 0.91 0.78 0.00 FS/N
      82   54   64   73   79   81   81   80   74   64 -999 SNxx
 
 18 21.2  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  8.3 19.1  24.4
     1F2  2E   3F2  2F2  2F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      13    7   30   19   24    6    7    8   13   14    0 ANGL
      67   52   56   63   63   67   66   67   62   51 -999 DBU
      53   48   51   55   54   56   53   53   48   35 -999 dBpW
     101   78   83   91   93   97   96   99   96   85 -999 S/N
    0.99 0.70 0.82 0.95 0.97 0.99 0.99 0.98 0.95 0.82 0.00 FS/N
      83   63   68   77   79   83   81   81   78   67 -999 SNxx
 
 19 21.3  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  6.0 19.2  24.5
     1F2  2E   2F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      13    7   16    4    5    6    7    9   13   14    0 ANGL
      68   63   65   69   69   69   68   68   63   51 -999 DBU
      54   59   60   60   58   57   55   54   49   35 -999 dBpW
     102   88   92   95   97   99   98  100   97   86 -999 S/N
    0.99 0.92 0.95 0.99 0.99 0.99 0.99 0.98 0.96 0.83 0.00 FS/N
      84   74   77   82   84   85   83   82   79   68 -999 SNxx
 
 20 20.6  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  3.3 18.5  23.6
     1F2  2F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      14   15    4    4    5    7    8   10   14   15    0 ANGL
      68   68   70   71   70   70   69   68   60   50 -999 DBU
      54   65   63   61   59   58   56   55   46   34 -999 dBpW
     102   93   94   97   99   99   99  101   94   85 -999 S/N
    0.99 0.98 0.99 0.99 0.99 0.99 0.99 0.99 0.93 0.81 0.00 FS/N
      84   80   81   84   85   86   84   83   76   66 -999 SNxx
CCIR Coefficients  METHOD  6   REC533 Version 16.1207a             PAGE   5
                                                  Analog modulation
 JUN    1994          SSN = 100.                      Path
 TANGIER, Morocco    BELGRADE               AZIMUTHS <Short> N. MI.      KM
 35.80 N    5.90 W   44.90 N   20.50 E     57.41  254.73    1318.1   2440.9
 MIN ANG  3.0 DEG
 XMTR  2-30 2-D Table [DEFAULT\CONST17.VOA  ] Az=  0.0 OFFaz= 57.4 500.000kW
 RCVR  2-30 2-D Table [DEFAULT\SWWHIP.VOA   ] Az=  0.0 OFFaz=254.7
 NOISE -145 dBW           S/N 90% of Days @  73 dB  in    1 Hz RX Bandwidth
 
 UT  MUF                                                         LUF  FOT OPMUF
 
 21 19.5  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  3.2 19.1  24.3
     1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      14    4    4    5    6    7    9   11   15   16    0 ANGL
      68   70   70   71   70   69   69   68   56   48 -999 DBU
      55   64   64   61   59   57   56   55   42   32 -999 dBpW
     102   93   94   97   99   99  100  101   90   83 -999 S/N
    0.99 0.98 0.99 0.99 0.99 0.99 0.99 0.99 0.89 0.79 0.00 FS/N
      84   79   81   84   85   85   85   83   72   66 -999 SNxx
 
 22 18.5  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  3.2 18.1  23.1
     1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      15    4    4    5    7    8   10   13   15   16    0 ANGL
      68   70   71   71   70   69   69   68   53   46 -999 DBU
      55   65   64   62   60   57   56   56   39   30 -999 dBpW
     102   93   95   98   99   99  100  102   88   81 -999 S/N
    0.99 0.98 0.99 0.99 0.99 0.99 0.99 0.99 0.86 0.75 0.00 FS/N
      84   80   81   85   86   86   83   84   69   64 -999 SNxx
 
 23 17.5  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  3.2 17.2  21.9
     1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      15    4    4    6    7    9   11   15   16   17    0 ANGL
      68   70   71   71   69   69   69   64   51   43 -999 DBU
      56   65   64   62   59   57   56   52   37   28 -999 dBpW
     102   93   95   98   99  100  101   98   85   79 -999 S/N
    0.99 0.98 0.99 0.99 0.99 0.99 0.99 0.97 0.82 0.70 0.00 FS/N
      84   80   82   85   85   86   83   80   67   62 -999 SNxx
 
 24 16.5  6.1  7.2  9.7 11.9 13.7 15.4 17.7 21.6 25.9  0.0 FREQ  3.1 16.1  20.6
     1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  1F2  0   MODE
      15    4    5    6    8   10   13   16   17   18    0 ANGL
      68   70   71   70   69   69   69   58   48   41 -999 DBU
      57   65   64   61   59   57   57   46   34   26 -999 dBpW
     102   94   96   98   99  100  101   92   83   76 -999 S/N
    0.99 0.98 0.99 0.99 0.99 0.99 0.99 0.92 0.80 0.64 0.00 FS/N
      84   80   82   85   85   84   83   74   66   61 -999 SNxx
 End-of-Run
